frontend/src/components/common/LogViewer.tsx

```diff
@@ -254,7 +254,7 @@ export function SearchPopover(props: SearchPopoverProps) {
   const [regexChecked, setRegexChecked] = React.useState<boolean>(false);
   const { t } = useTranslation(['translation']);
   const focusedRef = React.useCallback(
-    node => {
+    (node: HTMLInputElement) => {
```

Comment:
This and similar type changes could be done in a separate commit/PR.

Reply:
done